cumulus/parachains/common/src/lib.rs

```diff
@@ -109,8 +112,21 @@ mod constants {
 		polkadot_primitives::MAX_POV_SIZE as u64,
 	);
 
+	/// We allow for 2 seconds of compute with a 6 second average block.
+	pub const MAXIMUM_BLOCK_WEIGHT_FOR_ASYNC_BACKING: Weight = Weight::from_parts(
+		WEIGHT_REF_TIME_PER_SECOND.saturating_mul(2),
+		polkadot_primitives::MAX_POV_SIZE as u64,
+	);
+
 	/// Treasury pallet id of the local chain, used to convert into AccountId
 	pub const TREASURY_PALLET_ID: PalletId = PalletId(*b"py/trsry");
+
+	/// Maximal asset hub header size.
+	pub const MAX_ASSET_HUB_HEADER_SIZE: u32 = 4_096;
+	
+	/// Maximal bridge hub header size.
+	pub const MAX_BRIDGE_HUB_HEADER_SIZE: u32 = 4_096;
```

Comment:
These are bridge specific. If we keep `bp-bridge-hub-cumulus`, let's keep them there.

Reply:
Moved them back.